### ISSUE TITLE
Trouble getting global result options to work

### ISSUE THREAD

**Issue by alexweissman** [dowjones/gulp-bundle-assets#79]
I'm having some difficulty getting result options to work globally in my gulpfile.  I've tried:

```
        .pipe(plugins.bundleAssets.results({
            dest: './',  // destination of bundle.result.json
            type: {
              scripts: 'plain',
              styles: 'plain'
            }
        })) 
```

(where `plugins.bundleAssets` is a reference to the package).  It is still defaulting to the `html` format for results.

I've also tried using the syntax mentioned briefly in #9, though I suspect that was just pre-planning.

**Comment by chmontgomery** [comment]
Those `result.type` options are defined at the bundle level, not the global level. Have you tried following this example? https://github.com/dowjones/gulp-bundle-assets/tree/master/examples/custom-result#built-in

**Comment by alexweissman** [comment]
Yup - but those appear to be per-bundle, correct?  I was hoping to configure my task globally.

**Comment by chmontgomery** [comment]
Right. At this point that config is only available at the bundle level. I'd
accept a PR ;)
On Apr 3, 2016 9:58 AM, "Alex Weissman" <notifications@github.com> wrote:

Yup - but those appear to be per-bundle, correct? I was hoping to configure
my task globally.

—
You are receiving this because you commented.

Reply to this email directly or view it on GitHub
<https://github.com/dowjones/gulp-bundle-assets/issues/79#issuecomment-204993058>

**Comment by alexweissman** [comment]
Ah, gotcha.  Unfortunately, I'm pretty new to Node and have my hands full with UserFrosting.  Maybe I can find someone in our community who'd be willing to contribute :-D
